accessibility-checker-engine/help/IBMA_Color_Contrast_WCAG2AA_PV.mdx

```diff
@@ -12,19 +12,19 @@ import { CodeSnippet, Tag } from "carbon-components-react";
 
 * People with moderately low vision who do not use a contrast-enhancing technology 
 * People who do not see the full range of colors 
-* People who see no color. 
+* People who see no color
 
 ## Why is this important?
 
-When the foreground text and its background colors have less than a 4.5:1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology.
+When text and its background colors have less than a 4.5 to 1 contrast ratio it can be difficult for people with moderately low vision to read the text without a contrast-enhancing technology. For larger text of 18 point or more, or bold 14 point text, the text and background colors must give at least a 3 to 1 contrast ratio.
```

Comment:
When text and its background colors have less than a 4.5 to 1 contrast ratio it can be difficult for people with **moderate** low vision to read the text without a contrast-enhancing technology. For larger text of 18 **points** or more, or bold 14 point text, the text and background colors must give at least a 3 to 1 contrast ratio.

Reply:
I'd argue that 'moderately' (an adverb applying to the adjective 'low') is better here than 'moderate' (an adjective applying to 'low vision'), because we really mean to say that the level of vision is 'moderately low'.

i agree with Erick that 'points' is better than 'point'.